### PR TITLE
Added virtual scroll guide for cookbook

### LINKED ISSUE #1820: [Enhancement] Create virtual scroll guide for cookbook
- [x] I have written a [good issue](https://github.com/kirbydesign/designsystem/wiki/The-Good%3A-Issue) 

## Describe the enhancement
Once the restructuring of lists is done, in part to enable virtual scrolling, we should consider to make a virtual scroll guide on the cookbook instead of locking down the functionality in Kirby, as it will be hard to cover all possible use-cases.

This example should use cdk-virtual-scroll for angular.

## Describe the solution you'd like
A cookbook page (e.g. under the Layout Recipes menu, possibly renaming it to something like 'Guides')  with a virtual scroll guide showcasing a kirby-list built declaratively with content projection (slottet items).

The page could be constructed like the one for [item-sliding](https://cookbook.kirby.design/#/home/showcase/item-sliding), with a simple list that has virtual scroll enabled, using the simplest possible solution for virtual scroll, e.g. like the code snippets in Ionics example: https://ionicframework.com/docs/angular/virtual-scroll 

## Are there any additional context?

<hr />

## Checklist:

The following tasks should be carried out in sequence in order to follow [the process of contributing](https://github.com/kirbydesign/designsystem/blob/master/.github/CONTRIBUTING.md/#the-process-of-contributing) correctly.

### Refinement

- [ ] Request that the issue is [UX refined](https://github.com/kirbydesign/designsystem/blob/master/.github/CONTRIBUTING.md/#ux-refinement); do not proceed until this is done.
- [ ] Request that the issue is [tech refined](https://github.com/kirbydesign/designsystem/blob/master/.github/CONTRIBUTING.md/#tech-refinement); do not proceed until this is done.

### Implementation 
The contributor who wants to implement this issue should: 

- [x] Make sure you have read: "[Before you get coding](https://github.com/kirbydesign/designsystem/blob/master/.github/CONTRIBUTING.md/#before-you-get-coding)".
- [x] Signal to others you are working on the issue by assigning yourself.
- [x] Create a branch from the [master branch](https://github.com/kirbydesign/designsystem/tree/master) following our [branch naming convention](https://github.com/kirbydesign/designsystem/wiki/The-Good%3A-Branch). 
- [x] Publish a WIP implementation to Github as a draft PR and ask for feedback. 
- [x] Make sure you have implemented tests following the guidelines in: "[The good: Test](https://github.com/kirbydesign/designsystem/wiki/The-Good%3A-Test)".
- [x] Update the [cookbook](https://cookbook.kirby.design) with examples and showcases.

### Review
Once the issue has been implemented and is ready for review:

- [x] Do a [self-review](https://github.com/kirbydesign/designsystem/wiki/The-Good%3A-Self-review). 
- [x] Create a pull-request. If you created a draft PR during implementation you can just mark that as "ready for review".

### PR DESCRIPTION
**Pull request by mictro**
## Which issue does this PR close?

This PR closes #1820 

## What is the new behavior?

The 'Layout recipes' examples page has been renamed to 'Guides' and a guide for Virtual Scroll with a simple List has been added.  

## Does this PR introduce a breaking change?

- [ ] Yes
- [X] No

## Checklist:

The following tasks should be carried out in sequence in order to follow [the process of contributing](https://github.com/kirbydesign/designsystem/blob/master/.github/CONTRIBUTING.md/#the-process-of-contributing) correctly.

### Reminders
- [x] Make sure you have implemented tests following the guidelines in: "[The good: Test](https://github.com/kirbydesign/designsystem/wiki/The-Good%3A-Test)".
- [x] Make sure you have updated the cookbook with examples and showcases (for bug fixes, enhancements & new components).

### Review  
- [x] Do a [self-review](https://github.com/kirbydesign/designsystem/wiki/The-Good%3A-Self-review).
- [x] Request that the changes are code-reviewed 
- [x] Request that the changes are [UX reviewed](https://github.com/kirbydesign/designsystem/blob/master/.github/CONTRIBUTING.md/#ux-review) (only necessary if your PR introduces visual changes)

When the pull request has been approved it will be automatically merged to master via [automerge](https://docs.github.com/en/github/collaborating-with-issues-and-pull-requests/automatically-merging-a-pull-request).

### CODE REVIEW

**Comment by MadsBuchmann** [comment]
Haven't reviewed - just a quick thought 😄 

It does not seem like this touches any `libs/**` files which means it should trigger a new release of the Kirby package. 
So this one is perhaps fine to merge directly to the `master` branch.

File: apps/cookbook/src/app/examples/virtual-scroll-example/virtual-scroll-list-example/virtual-scroll-list-example.component.html [hunk 1]
```diff
@@ -0,0 +1,16 @@
+<kirby-page title="Items">
+  <kirby-page-content>
+    <cdk-virtual-scroll-viewport
+      minBufferPx="280"
+      maxBufferPx="560"
+      itemSize="56"
+      style="height: 280px;"
```
Comment (RasmusKjeldgaard) [inline apps/cookbook/src/app/examples/virtual-scroll-example/virtual-scroll-list-example/virtual-scroll-list-example.component.html:7]: should be bump the height a bit in this example, so it shows a bit more items? Something like 720px or so.

File: apps/cookbook/src/app/examples/virtual-scroll-example/virtual-scroll-list-example/virtual-scroll-list-example.component.scss [hunk 2]
```diff
@@ -0,0 +1,8 @@
+@use '~@kirbydesign/core/src/scss/utils';
+
+:host {
+  display: block;
+  width: 100%;
```
Comment (RasmusKjeldgaard) [inline apps/cookbook/src/app/examples/virtual-scroll-example/virtual-scroll-list-example/virtual-scroll-list-example.component.scss:5]: Very minor thing, but I don't believe you need this width here.

**Review by RasmusKjeldgaard — COMMENTED** [review]
Looks good, very nice work.
Some small comments for the example is all I have, while we are awaiting #2029

File: apps/cookbook/src/app/examples/virtual-scroll-example/virtual-scroll-list-example/virtual-scroll-list-example.component.html (hunk 1 above)
Comment (mictro) [reply apps/cookbook/src/app/examples/virtual-scroll-example/virtual-scroll-list-example/virtual-scroll-list-example.component.html]: Done

File: apps/cookbook/src/app/examples/virtual-scroll-example/virtual-scroll-list-example/virtual-scroll-list-example.component.scss (hunk 2 above)
Comment (mictro) [reply apps/cookbook/src/app/examples/virtual-scroll-example/virtual-scroll-list-example/virtual-scroll-list-example.component.scss]: Done

**Pull request merged**